macros/edr/dbt_artifacts/upload_dbt_columns.sql

```diff
@@ -0,0 +1,67 @@
+{%- macro upload_dbt_columns(should_commit=false, metadata_hashes=none) -%}
+    {% set relation = elementary.get_elementary_relation('dbt_columns') %}
+    {% if execute and relation %}
+        {% set tables = graph.nodes.values() | list + graph.sources.values() | list %}
+        {% do elementary.upload_artifacts_to_table(relation, tables, elementary.flatten_table_columns, should_commit=should_commit, metadata_hashes=metadata_hashes) %}
+    {%- endif -%}
+    {{- return('') -}}
+{%- endmacro -%}
+
+{% macro get_dbt_columns_empty_table_query() %}
+    {% set columns = [
+        ('unique_id', 'string'),
+        ('parent_unique_id', 'string'),
+        ('name', 'string'),
+        ('data_type', 'string'),
+        ('tags', 'long_string'),
+        ('meta', 'long_string'),
+        ('database_name', 'string'),
+        ('schema_name', 'string'),
+        ('table_name', 'string'),
+        ('description', 'long_string'),
+        ('generated_at', 'string'),
+        ('metadata_hash', 'string'),
+    ] %}
+    {% set dbt_columns_empty_table_query = elementary.empty_table(columns) %}
+    {{ return(dbt_columns_empty_table_query) }}
+{% endmacro %}
+
+{% macro flatten_table_columns(table_node) %}
+    {% set column_nodes = table_node.get("columns") %}
+    {% if not column_nodes %}
```

Comment:
Should we not include columns that don't have descriptions for optimization's-sake?

Reply:
I'm in favor of including all of them unless we see it causes a performance issue